Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/DocumentPropertyEditor.cpp

```diff
@@ -475,6 +492,12 @@ namespace AzToolsFramework
         for (auto operationIterator = patch.begin(), endIterator = patch.end(); operationIterator != endIterator; ++operationIterator)
         {
             const auto& patchPath = operationIterator->GetDestinationPath();
+            if (patchPath.Size() == 0)
```

Comment:
makes sense, but when can this happen?

Reply:
This can happen if a top-level replace patch occurs replacing the entirety of the adapter's contents. Before I changed the patch detection threshold I was actually getting a crash because of this, and top-level patches are full valid.